internal/pkg/instrumentation/manager.go

```diff
@@ -199,23 +199,23 @@ func (m *Manager) load(target *process.TargetDetails) error {
 
 	// Load probes
 	for name, i := range m.probes {
-		m.logger.Info("loading probe", "name", name)
+		m.logger.V(-1).Info("loading probe", "name", name)
```

Comment:
I think this might be useful to keep as info, to know which probes are activated.

Reply:
Changed to info